docs/comparison/nn_tracking.md

```diff
@@ -0,0 +1,180 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Real-time tracking of Neural Network training
+
+SQLiteTracker provides a powerful and flexible way to track computational (e.g., Machine Learning) experiments using a SQLite database.
+
+In this tutorial we demonstrate training a small network on the Fashion MNIST dataset and tracking the training metrics in realtime. We would also see how you can query the tracked metrics while training is going on, and visualise the metrics vs epoch plots.
+
++++
+
+## Create the experiment tracker
+
+```{code-cell} ipython3
+from pathlib import Path
+from sklearn_evaluation import SQLiteTracker
+
+db = Path('nn_experiments.db')
+
+if db.exists():
+    db.unlink()
```

Comment:
let's put this in a cell above and add the `:hide-cell:` tag, so it doesn't show by default https://jupyterbook.org/en/stable/interactive/hiding.html#hide-entire-code-cells